app/views/passwords/new.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      = form_tag passwords_path, method: :post do
+        %header
+          %h1 Forgot Password?
+          %p We will send you an email with a link that you can use to reset your password.
+        .form-group
+          = label_tag :email, "Email Address"
+          .row
+            .col-sm-4
+              = text_field_tag :email, params[:email], type: "email",class: "form-control"
```

Comment:
I would rather use `email_field_tag` here. No need to set the `type: "email"` then.